config/samples/datadog-operator-hub-example.yaml

```
@@ -0,0 +1,43 @@
+apiVersion: datadoghq.com/v1alpha1
+kind: DatadogAgent
+metadata:
+  name: datadog
+spec:
+  clusterName: <CLUSTER_NAME>
+  credentials:
+    apiKey: <DATADOG_API_KEY>
+    appKey: <DATADOG_APP_KEY>
+    token: <DATADOG_CLUSTER_AGENT_TOKEN>
+  agent:
+    image:
+      name: "datadog/agent:7.24.1"
```

Should we switch to the gcr images instead ?
```suggestion
      name: "gcr.io/datadoghq/agent:7.24.1"
```